pyas2/models.py

```diff
@@ -19,6 +19,10 @@
 from pyas2.utils import run_post_send
 from pyas2.utils import store_file
 
+from pyas2lib.utils import pem_to_der
+from OpenSSL import crypto
```

Comment:
Please add pyopenssl to the requirements and setup file

Reply:
Added, however, that library already installed through pyas2lib.